src/screens/Survey/components/CustomInput/index.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
This can be avoided using `htmlFor='inputId'` in the label and `id='inputId'` in the input itself

Reply:
Further reading: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md